download.html

```diff
@@ -285,29 +285,72 @@ <h1>Apache Ignite <strong>Downloads</strong></h1>
 
         <section id="toc" class="page-section">
-            <div class="row">
-                <div class="col-sm-6">
-                    <ul>
-                        <li><a href="#sources">Source Releases</a></li>
-                        <li><a href="#binaries">Binary Releases</a></li>
-                        <li><a href="#nightly-builds">Nightly Builds</a></li>
-                        <li><a href="#docker">Docker and Cloud Images</a></li>
-                        <li><a href="#maven">Download from Maven</a></li>
-                        <li><a href="#nuget">Download from NuGet</a></li>
-                        <li><a href="#rpm-package">RPM Package Installation</a></li>
-                        <li><a href="#deb-package">DEB Package Installation</a></li>
-                        <li><a href="#build-source">Build From Source</a></li>
-                        <li><a href="#git-repository">Git Repository</a></li>
-                        <li><a href="#3rdparty">3<sup>rd</sup> Party Binaries</a></li>
-                    </ul>
+            <div class="jumbotron jumbotron-fluid">
+                <div class="container">
+                    <div class="title display-6">Learn More</div>
+                    <hr class="my-4">
+                    <div class="row">
+                        <div class="col-sm-6">
+                            <ul>
+                                <li><a href="#sources">Source Releases</a></li>
+                                <li><a href="#binaries">Binary Releases</a></li>
+                                <li><a href="#build-source">Building the Binaries</a></li>
+                                <li><a href="#docker">Docker and Cloud Images</a></li>
+                                <li><a href="#git-repository">Git Repository</a></li>
+                            </ul>
+                        </div>
+                        <div class="col-sm-6">
+                            <ul>
+                                <li><a href="#maven">Download from Maven</a></li>
+                                <li><a href="#nuget">Download from NuGet</a></li>
+                                <li><a href="#deb-package">DEB Package Installation</a></li>
+                                <li><a href="#rpm-package">RPM Package Installation</a></li>
+                                <li><a href="#3rdparty">3<sup>rd</sup> Party Binaries</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <p>
+                        To get LGPL dependencies (e.g. Hibernate L2 Cache),
+                        see <a href="#build-source">Building The Binaries</a>.</p>
                 </div>
             </div>
-            <p>* To get LGPL dependencies (e.g. Hibernate L2 Cache), see <a href="#build-source">Building The
-                Binaries</a>.</p>
+            <p>
+                If you encounter a problem with the selected mirror, please select another mirror.
```

Comment:
Please update the paragraph to the one below and I would place it under the list (not before).

_If you encounter a problem with the selected mirror, please choose another one. If primary mirrors are not reachable, switch to <i>backup</i> servers added to the end of the list._

Reply:
Denis, I've changed the sentence and move it down.

But from my point, it's better to keep it on top to not force uses to scroll the whole page down and read all notes if they encounter a problem with download releases.

Reply:
If we place the sentence right below the mirrors' selection list then folks will see it if they decide to change the mirror. They should change the mirror only in case of an issue is encountered, thus they will see the message anyway. By default, the website will pre-select the mirror closest to the location of a user.

Just don't want us to distract people with an emergency text by placing it at the top of the downloads list ;)